rdkit_valgrind.suppressions

```diff
@@ -0,0 +1,3235 @@
+{
+   uninitialised value used in logic 4
+   Memcheck:Cond
+   fun:__mpn_addmul_1
+   obj:*
+   obj:*
+   fun:__mpn_mul
+   obj:*
+   obj:*
+   obj:*
+   obj:*
+   obj:*
+   obj:*
+   obj:*
+   fun:__printf_fp
+   obj:*
```

Comment:
The trailing obj:* add nothing and only prevent shorter call stacks from benefitting from these suppressions.  (the ones between __mpn_mul and __printf_fp are meaningful).

Also, I know that you used the internal valgrind suppression deduplicator - did you check that it's not starring something that is meaningful in an RDKit context?

Reply:
I'll get rid of the trailing ones. The deduplicator is "starring" any objs starting with a slash... which seems like all of them.

Reply:
oh, yeah. I think that is because in our builds, objects with relative paths are in the build, and absolute paths are outside of the build (in shared libraries). Furthermore, suppressions files with absolute paths aren't relocatable for running the tests a second time.

This probably means that these suppressions files are overly general. It would probably be best to replace $RD_PATH with a * and leave the rest of the path/object name explicit.